setup.py

```diff
@@ -84,14 +80,11 @@ def read(*names, **kwargs):
         "cdsapi >= 0.1.4",
         "geopandas",
         "numpy >= 1.17.0",
-        "oedialect >= 0.0.6.dev0",
```

Comment:
I think the oedialect is needed to download open_FRED weather data. Shouldn't we keep this as a requirement? I see the weather data download as an essential functionality of the feedinlib.